routes/voyager.php

```diff
@@ -13,15 +15,15 @@
 */
 
 Route::group(['as' => 'voyager.'], function () {
-    event('voyager.routing', app('router'));
+    event(new Routing());
```

Comment:
Not sure what these events are even for... @tnylea / @marktopper, thoughts?

Reply:
After a search on my local packages, I've found an hint with Hooks.

Hooks setup will be revised, me and @marktopper have a draft for Hooks also based on events, when we check that, maybe we can remove this event? 
Do you want to remove it Fletch? I don't know if is that heavy for a system firing events.

Reply:
Its used by the hook system and some other hooks from what I know. Maybe we can remove this in `develop`.